app/Resources/views/base_angular.html.twig

```diff
@@ -11,13 +11,11 @@
 {% block javascripts %}
     <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.4.7/angular.min.js"></script>
     <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.4.7/angular-route.min.js"></script>
-    <script src="//cdn.rawgit.com/HackedByChinese/ng-idle/1.1.1/angular-idle.min.js"></script>
+    <script src="{{ asset('assets/lib/src/vendor/angular-idle.min.js') }}"></script>
```

Comment:
I had to update this to a newer version in order to fix a minifying issue. We can probably link back off to a CDN hosted one.